plugins/modules/homebrew_services.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Kit Ham <kitizz.devside@gmail.com>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+import json
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: homebrew_services
+author:
+    - "Kit Ham (@kitizz)"
+requirements:
+    - homebrew must already be installed on the target system
+short_description: Services manager for Homebrew
+description:
+    - Manages daemons and services via Homebrew
+extends_documentation_fragment:
+    - community.general.attributes
+options:
+    name:
+        description:
+            - An installed homebrew package whose service is to be updated.
+        aliases: [ 'formula', 'package', 'pkg' ]
+        type: str
+    path:
+        description:
+            - "A V(:) separated list of paths to search for C(brew) executable.
+              Since a package (I(formula) in homebrew parlance) location is prefixed relative to the actual path of C(brew) command,
+              providing an alternative C(brew) path enables managing different set of packages in an alternative location in the system."
+        default: '/usr/local/bin:/opt/homebrew/bin:/home/linuxbrew/.linuxbrew/bin'
+        type: path
+    state:
+        description:
+            - state of the package.
+        choices: [ 'present', 'absent', 'restarted' ]
+        default: present
+        type: str
+"""
+
+EXAMPLES = """
+# Install foo package
+- community.general.homebrew:
+    name: foo
+    state: present
+
+# Start the foo service (equivalent to C(brew services start foo))
```

Comment:
```suggestion
# Start the foo service (equivalent to 'brew services start foo')
```
Formatting does not work in the examples, this will be shown to the user verbatim.

Reply:
Gah, you can tell I didn't read the docs about when to use that formatting style